geofront/server.py

```diff
@@ -1033,6 +1042,58 @@ def authorize_remote(token_id: str, alias: str):
     )
 
 
+@sockets.route('/ws/tokens/<token_id:token_id>/remotes/<alias>/ssh/')
+def proxy_ssh(ws, token_id: str, alias: str):
```

Comment:
Added, but not shown in the docs currently due to kennethreitz/flask-sockets#50.